src/dev/riscv/LupV.py

```diff
@@ -0,0 +1,182 @@
+from m5.objects.Platform import Platform
+from m5.objects.PMAChecker import PMAChecker
+from m5.objects.Clint import Clint
+from m5.objects.LupioRTC import LupioRTC
+from m5.objects.Plic import Plic
+from m5.objects.RTC import RiscvRTC
+from m5.objects.Uart import Uart8250
+from m5.objects.Terminal import Terminal
+from m5.params import *
+from m5.proxy import *
+from m5.util.fdthelper import *
```

Comment:
Again, double check that each import is necessary.

Reply:
I double checked the imports and am now wondering if it is ok to have the `import *` for `params` and `fdthelper`?

Reply:
Seems fine to me :)